src/cards/share.scss

```diff
@@ -179,7 +179,7 @@
 .share--left .share__shares,
 .share--gmt .share__shares {
   margin-left: 0.67in;
-  padding: 0.125in;
+  padding: 0.0625in 0.125in;
```

Comment:
Testing this on http://18xx-maker.com/ results in not enough bleed on the cards. Confused that something like that that I've used on MANY prints of games is now suddenly wrong?

It's also the case that if we DID want to make this change, `padding: 0.0625in 0.125in;` is the default on `.share__precent` so we can just remove this override, instead of changing it.

Reply:
OK, I didn't notice that.  What bleed is it affecting?  It was just showing extra space at the bottom that pushed the "1 share" and "10%" up away from the bottom edge more than it needed to be, and more than the other edges, making the card look like I'd cut if off center.

Feel free to drop this part of the PR or let me know if I need to address it and re-push/PR.

Reply:
So, here is what I'm seeing.  This is before this change:
<img width="288" alt="Screen Shot 2021-01-28 at 10 53 26 PM" src="https://user-images.githubusercontent.com/48138042/106237308-c19cf600-61bb-11eb-9eb2-992a1492ecf6.png">
I've connected the cut lines to show the actual corner.  Note that the 10% sits high in its corner.  After my change:
<img width="294" alt="Screen Shot 2021-01-28 at 10 45 21 PM" src="https://user-images.githubusercontent.com/48138042/106237396-f9a43900-61bb-11eb-986d-b8347d7a3d4c.png">
Now the bottom right corner of the 10% is properly at a 45° angle from the card corner.

It's not much, but the original being high in the corner for its horizontal position looks off, and that bugged me.  Thus, this fix.  If there's a correct way to fix this, let me know.

Reply:
Yeah I think the problem is that the value SHOULD be the original one and we have text that isn't rendering at that base line, so we're compensating for where the text renders relative to the bottom but raw placement. I'd rather fix this by making the text render on the bottom and using the proper padding value, but I don't know if I have time to look into it anytime soon :-( We can just pull this in and if I get to fix it right I can do that later.